### PR TITLE
feat: add java options and recommended/default values

### PR DESCRIPTION
**Pull request by infojohn**
solves PZ-4056 (partially)

### CODE REVIEW

**Comment by hsiliev** [comment]
Perhaps we can merge this PR with https://github.com/infonl/dimpact-zaakafhandelcomponent/pull/1794

File: helm/templates/config.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- include "zaakafhandelcomponent.labels" . | nindent 4 }}
 data:
+  _JAVA_OPTIONS: {{ .Values.javaOptions | default "-Xmx1024m -Xms1024m -Xlog:gc::time,uptime" | quote }}
```
Comment (hsiliev) [inline helm/templates/config.yaml:8]: That would reset the current 1700MB to 1024MB right?

Comment (infojohn) [reply helm/templates/config.yaml:8]: correct, that would have to be set via the values then

**Review by bas-info-nl — APPROVED** [review]
But where do we set the actual javaOptions value?

**Comment by hsiliev** [comment]
> But where do we set the actual javaOptions value?

This should be deployment specific and part of the infra settings:
* local dev - 1GB
* test - 2GB

So for dev the settings would be in docker compose, while for test this would be helm chart.

**Review by hsiliev — APPROVED** [review]

File: helm/values.yaml [hunk 2]
```diff
@@ -144,6 +144,9 @@ maxFileSizeMB: 80
 # remoteDebug Enable Java remote debugging
 remoteDebug: False
 
+# javaOptions: jvm options to set in the _JAVA_OPTIONS env var, recommended defaults are used if not provided and are shown below
+#javaOptions: -Xmx1024m -Xms1024m -Xlog:gc::time,uptime
```
[the remark this answers is not in the document]
Comment (infojohn) [reply helm/values.yaml:148]: I have created https://github.com/infonl/dimpact-provisioning/pull/127 to add the jvm options from the github env vars

[the remark this answers is not in the document]
Comment (hsiliev) [reply helm/values.yaml:148]: And as we discussed the real value for test is in GitHub environment variables.

**Comment by hsiliev** [comment]
Removed explicitly set Dockerfile memory setting

**Pull request merged**